### DISCUSSION TITLE
Long response times and errors after upgrade to 4.0.8

### DISCUSSION THREAD

**Discussion by d1nuc0m** [netbox-community/netbox#17014] [Help Wanted!]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.8

### Python Version

3.12

### Steps to Reproduce

1. Upgraded NetBox 3.7.4 --> 3.7.8 --> 4.0.8 (self-hosted on EL9), no errors reported. Used `sudo PYTHON=/usr/bin/python3.12 /opt/netbox/upgrade.sh`
2. Users complained of very long response times, it takes > 45 seconds to draw Rack Elevations with 7 racks
3. Enabled logging [as here](https://majornetwork.net/2022/09/configuring-logging-in-netbox/)

Logging settings:
```python
LOGGING = {
    'version': 1,
    'disable_existing_loggers': False,
    'formatters': {
        'normal': {
            'format': '%(asctime)s %(name)s %(levelname)s: %(message)s'
        },
    },
    'handlers': {
        'file': {
            'level': 'DEBUG',
            'class': 'logging.handlers.WatchedFileHandler',
            'filename': '/var/log/netbox/netbox.log',
            'formatter': 'normal',
        },
    },
    'loggers': {
        'django': {
            'handlers': ['file'],
            'level': 'DEBUG',
        },
        'netbox': {
            'handlers': ['file'],
            'level': 'DEBUG',
        },
    },
}
```

Local requirements
```
netbox-ipcalculator
netbox-lifecycle
dulwich
django-auth-ldap
netbox-proxbox
```

### Expected Behavior

NetBox continues working as before the upgrade

### Observed Behavior

Every action results in long loading times and tons of errors, especially for Rack Elevations, log attached: [netbox.log](https://github.com/user-attachments/files/16413885/netbox.log)

**Comment by candlerb** [comment]
What browser are you using? Have you tried a different browser? Some people are complaining of slowness with Safari only.

What exactly did you configure to get those logs? Netbox catches exceptions as part of its normal operation.

**Comment by d1nuc0m** [comment]
> What browser are you using? Have you tried a different browser?

Firefox and Chromium, on Windows 11 and EL9

> What exactly did you configure to get those logs?

`/opt/netbox/netbox/netbox/configuration.py` here: [configuration.txt](https://github.com/user-attachments/files/16422939/configuration.txt)
